custom/icds/templates/icds/messaging/indicators/urd/aww_aggregate_performance.txt

```diff
@@ -0,0 +1,10 @@
+آپ کے آنگن واڑی مرکز کیلئے گذشتہ 30 دنوں میں مجموعی کارکردگی کی رپورٹ
+
+1. دوروں کی تعداد / مطلوبہ دوروں کی تعداد:{{ visits }} / 65
```

Comment:
This is incorrect structure. The{{ visits }}/ 65 should be at the starting as Urdu is read from right to left. Similarly for all the other conditions

Is there a reason why we modified this @dungeonmaster51 ? 

FYI @mkangia

Reply:
I verified and this happens when we copy paste it. Verified it  on other places in the app. It is written this way only. No action needed on this.